libs/cardano-ledger-api/cardano-ledger-api.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-api
-version:            1.7.0.1
+version:            1.8.0.0
```

Comment:
We need this entry in the changelog for `cardano-ledger-api` and a change of the version too:
```markdown
* Change the type of `ppEMaxL`, `ppuEMaxL`, `ppGovActionLifetimeL`, `ppDRepActivityL`,
  `ppCommitteeMaxTermLengthL`, `ppuGovActionLifetimeL`, `ppuDRepActivityL` and 
  `ppuCommitteeMaxTermLengthL`
```

Reply:
done